alerta/database/base.py

```diff
@@ -29,8 +30,15 @@ def get_backend(app):
 
 
 def load_backend(backend):
+    for ep in iter_entry_points('alerta.backends'):
```

Comment:
Should this be `iter_entry_points('alerta.database.backends')` to be consistent with default db backend location?

Reply:
Can you add a minimal example db backend and maybe add it to `examples/backends`?